content/kotsadm/installing/helm-chart.md

```diff
@@ -0,0 +1,123 @@
+---
+date: 2019-10-09
+linktitle: "Installing a Helm Chart"
+title: Installing a Helm Chart
+weight: 10030
+draft: false
+---
+
+The most direct and simple way to install a KOTS application to a Kubernetes cluster is to deploy to an existing cluster that contains nodes that can access the Internet. In this scenario, the container images will be pulled from the upstream registries directly.
+
+**Note** Currently the only supported source for helm charts are public helm repositories support by helm out-of-the-box.
```

Comment:
I'm not sure if this is true or not, but I could be convinced to omit it either way.